src/components/calcite-tip-manager/calcite-tip-manager.scss

```diff
@@ -59,35 +59,31 @@
 
 /* Pagination Styles*/
 .pagination {
-  display: flex;
-  align-items: center;
-  justify-content: center;
-  padding: var(--calcite-spacing-quarter) 0;
+  @apply flex items-center justify-center py-1 px-0;
 }
 
 .page-position {
-  @include font-size(-4);
-  margin: 0 var(--calcite-spacing-half);
+  @apply text--2h my-0 mx-2;
```

Comment:
I think you replaced `-4` with `-2` here, just confirming that is intentional

Reply:
Yes, this is what is on the Figma design. Thanks for checking. Good eyes Paul, good eyes. You still got it.